crates/apollo-encoder/Cargo.toml

```diff
@@ -19,6 +19,14 @@ categories = [
 ]
 edition = "2021"
 
+[dependencies]
+apollo-parser = { version = "0.2.12", optional = true }
+thiserror = "1.0.37"
+
+[features]
+default = ["from-parser"]
+from-parser = ["apollo-parser"]
```

Comment:
should we just call this feature `apollo-parser`?

Reply:
i'm not sure what the general practice is on this in rustland 🤔 I used `from-parser` to describe what the feature actually does. but, if there are other features in the future that also rely on apollo-parser, it's fine if the TryFrom impls are made available anyways, so maybe that is actually counterproductive.

Reply:
I am leaning towards having this TryFrom be always available. It's part of the same ecosystem, so it makes sense.

Reply:
ok, I think then the current way is fine, as it's enabled by default but you can opt-out by setting `default-features=false` if you really only need the encoder and don't want the extra dependencies.

Reply:
decided to do tht^ in slack : )